src/bootstrap/src/core/config/config.rs

```diff
@@ -347,6 +347,7 @@ pub struct Config {
     pub ferrocene_document_signatures_s3_bucket: String,
     pub ferrocene_ignore_document_signatures: bool,
     pub ferrocene_technical_report_url: Option<String>,
+    pub ferrocene_code_coverage: bool,
```

Comment:
In the rest of the bootstrap codebase, the value of command line flags is not added to the `Config` struct, but as a method of the `Subcommand` struct (in `flags.rs`). See for example `Subcommand::bless()`.